CONTRIBUTING.md

```diff
@@ -74,7 +74,10 @@ Connect with us through https://chat.fhir.org/ or open an [issue](https://github
 ## Setup
 The IBM FHIR Server is built with Maven and requires Java 8 or higher.  To build the project from the root directory, please execute:
 
-> mvn clean install -f fhir-parent/pom.xml
+```
+mvn clean install -f fhir-examples
```

Comment:
Good point, happy to add

Reply:
I thought fhir-examples was now a dependency available in maven central so this step wouldn't be required?

Reply:
It depends ... if in the current release we are using the snapshot.

Reply:
Perhaps also worth pointing people over to the "setting up for development" wiki instead of having instructions twice which may get out of sync.